athena_cli.py

```diff
@@ -1,3 +1,4 @@
+#!/usr/bin/env python3
```

Comment:
Instead of adding a shebang here you should add "python_requires" option to `setup.py`. See https://packaging.python.org/guides/distributing-packages-using-setuptools/#python-requires

Reply:
Agreed. I'll check that in in a few

Reply:
I set the python version requirement to 3.5 and also set a cmd2 version requirement

Reply:
This line can be removed. It's ignored anyway as this is a python console script not a shell script.

Reply:
OK I removed it